interview-questions.md

```diff
@@ -4,58 +4,58 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: To "inheret" or take the properties and methods of another class.
 
-  Researched answer:
+  Researched answer: The super() method is used within the constructor of one class to call on the constructor of another class. 
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer: I don't know exactly, I now it helps React work. 
```

Comment:
Haha, totally appreciate the honesty here. You'll hear Bryan talk about this in PD week when practicing for interviews- but I would encourage you in an actual interview scenario that you would respond back honestly like this but to also show how you would find the information out. For example, instead of saying "I don't know exactly"- lets try saying "I know it helps React work. I'd need to take a little time and look into the documentation a bit to really give you a better answer." 

This shows that even though you may not know the answer RIGHT now, you are proactive in searching for the answer and know how/where to look to find the information you need to solve this problem you're presented with.